tests/test_comments.py

```diff
@@ -97,6 +98,15 @@ def test_user_comment_replies_are_none(self):
                              lambda item: isinstance(item, Comment))
         self.assertEqual(comment._replies, None)
 
+    @betamax
+    def test_unpickle_comment(self):
+        item = next(self.r.user.get_comments())
+        pkl = pickle.dumps(item)
+        try:
```

Comment:
This could be rewritten as

    assert_equal(item, pickle.loads(pickle.dumps(item)))

But I'm happy with what's there currently.